index.md

```diff
@@ -107,7 +107,7 @@ description: Vanilla OS is an Immutable Linux-based distribution which aims to p
             </a>
         </div>
         <div class="hero-image" data-scroll>
-            <img class="animate-on-scroll" data-animation="fadeInLeft" src="/assets/images/home-desk-apx.svg" alt="Vanilla OS - Immutable!" />
+            <img class="animate-on-scroll" data-animation="fadeInLeft" src="/assets/images/home-desk-apx.svg" alt="A simple terminal app showing the commands “apx install nice-stuff” and “abroot exec apt install nice-stuff” />
```

Comment:
Very nit, add a comment here that it's partially informative. Would help so it's not cut out in the future.